src/components/Trilemma/index.tsx

```diff
@@ -32,55 +32,46 @@ const Trilemma = () => {
     ...trilemmaChecks,
   }
 
-  // TODO: Replace with direct token implementation after UI migration is completed
-  const lgBp = useToken("breakpoints", "lg")
-
   return (
     <Flex
-      alignItems="flex-start"
+      alignItems="center"
       flexDirection={{ base: "column", lg: "row" }}
       justifyContent="space-between"
+      columnGap="12"
+      rowGap="8"
     >
-      <Flex
-        flexDirection="column"
-        my={{ base: 8, md: 12 }}
-        mx={{ md: 12 }}
-        py={8}
+      <Stack
+        spacing="8"
+        mt={{ base: "16", md: "20" }}
+        mb={{ lg: "20" }}
+        ms={{ md: "12" }}
+        me={{ md: "12", lg: 0 }}
         flex={{
-          base: "1 1 100%",
           lg: "0 1 500px",
         }}
       >
-        <OldHeading fontSize="2rem" mt={0}>
+        <Heading fontSize="2rem">
           {t("page-roadmap-vision-trilemma-h2")}
-        </OldHeading>
-        <Text>{t("page-roadmap-vision-trilemma-p")}</Text>
-        <Text>{t("page-roadmap-vision-trilemma-p-1")}</Text>
-        <Text>{t("page-roadmap-vision-trilemma-p-2")}</Text>
-        <Text fontWeight={600} hideFrom={lgBp}>
-          {t("page-roadmap-vision-trilemma-modal-tip")}:
-        </Text>
-        <Card {...cardDetail} mt={8} minH="300px" hideBelow={lgBp} />
-      </Flex>
-      <Hide above={lgBp}>
+        </Heading>
+        <Flex flexDir="column" gap="6">
+          <Text>{t("page-roadmap-vision-trilemma-p")}</Text>
+          <Text>{t("page-roadmap-vision-trilemma-p-1")}</Text>
+          <Text>{t("page-roadmap-vision-trilemma-p-2")}</Text>
+          <Text fontWeight={600} hideFrom="lg">
+            {t("page-roadmap-vision-trilemma-modal-tip")}:
+          </Text>
+        </Flex>
+        <Card {...cardDetail} minH="300px" hideBelow="lg" />
+      </Stack>
+      <Hide above="lg">
         <Drawer
           isOpen={mobileModalOpen}
           onClose={handleModalClose}
           placement="bottom"
         >
           <DrawerOverlay background="rgba(0,0,0,0.3)" />
-          <DrawerContent
-            border="none"
-            borderTopRadius="1rem"
-            background="background.base"
-          >
-            <Card
-              {...cardDetail}
-              background="none"
-              border="none"
-              justifyContent="flex-start"
-              my={8}
-            />
+          <DrawerContent borderTopRadius="1rem" background="background.base">
```

Comment:
Could we use `borderTopRadius="2xl"` here?

<img width="269" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/65b6cc9d-3782-4a32-851d-d4885c5106a3">

```suggestion
          <DrawerContent borderTopRadius="2xl" background="background.base">
```